<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TiffViewerClass</class>
 <widget class="QWidget" name="TiffViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TiffViewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">.QWidget{
border-radius: 8px;
background-color: rgb(232, 232, 232);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_icon">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLabel{
border-radius: 8px;
background-color: rgb(232, 232, 232);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="font">
         <font>
          <family>Segoe UI Variable</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">.QLabel{
border-radius: 8px;
background-color: rgb(232, 232, 232);
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
}</string>
        </property>
        <property name="text">
         <string>Tiff Viewer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>138</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QPushButton" name="pb_mini">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>46</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe Fluent Icons</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgba(0, 0, 0,0);
border:none;
color:rgb(0, 0, 0);
}
QPushButton:hover{
background-color: rgb(229, 229, 229);
}
QPushButton:pressed{
background-color: rgb(204, 204, 204);
}</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_exit">
         <property name="geometry">
          <rect>
           <x>92</x>
           <y>0</y>
           <width>46</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe Fluent Icons</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgba(0, 0, 0,0);
border-top-right-radius:8px;
border:none;
color:rgb(0, 0, 0);
}
QPushButton:hover{
background-color: rgb(232, 17, 35);
border:none;
	color: rgb(255, 255, 255);
}
QPushButton:pressed{
background-color: rgb(241, 112, 122);
border:none;
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
        <widget class="QPushButton" name="pb_max">
         <property name="geometry">
          <rect>
           <x>46</x>
           <y>0</y>
           <width>46</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe Fluent Icons</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgba(0, 0, 0,0);
border:none;
color:rgb(0, 0, 0);
}
QPushButton:hover{
background-color: rgb(229, 229, 229);
}
QPushButton:pressed{
background-color: rgb(204, 204, 204);
}</string>
         </property>
         <property name="text">
          <string></string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_client" native="true">
     <property name="styleSheet">
      <string notr="true">.QWidget{
border-radius: 8px;
background-color: rgb(240, 240, 240);
border-top-left-radius: 0px;
border-top-right-radius: 0px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_render" native="true">
        <property name="styleSheet">
         <string notr="true">.QWidget{
background-color: rgb(223, 223, 223);
border-radius: 8px;
}</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_bar" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">.QWidget{
background-color: rgb(250,250,250);
border-radius: 8px;
}
QPushButton{
background-color: rgb(255, 255, 255);
border: 1px solid rgb(240, 240, 240) ;
border-bottom-width: 2px;
border-radius: 4px;
}
QPushButton:hover{
background-color: rgb(240, 240, 240);
border: 1px solid rgb(225, 225, 225) ;
}

QPushButton:pressed{
background-color: rgb(215, 215, 215);
border: 1px solid rgb(230, 230, 230) ;
}
QPushButton:checked{
background-color: rgb(215, 215, 215);
border: 1px solid rgb(230, 230, 230) ;
}
</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_file" native="true">
           <property name="minimumSize">
            <size>
             <width>101</width>
             <height>0</height>
            </size>
           </property>
           <widget class="QPushButton" name="pb_open">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>81</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_back_forward" native="true">
           <property name="minimumSize">
            <size>
             <width>86</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>86</width>
             <height>16777215</height>
            </size>
           </property>
           <widget class="QPushButton" name="pb_left_back">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
            <property name="shortcut">
             <string>Left</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_right_forward">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>10</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="shortcut">
             <string>Right</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_zoom" native="true">
           <property name="maximumSize">
            <size>
             <width>86</width>
             <height>16777215</height>
            </size>
           </property>
           <widget class="QPushButton" name="pb_zoom_in">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <property name="shortcut">
             <string>Up</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_zoom_out">
            <property name="geometry">
             <rect>
              <x>45</x>
              <y>10</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <property name="shortcut">
             <string>Down</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_param" native="true">
           <property name="minimumSize">
            <size>
             <width>423</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>423</width>
             <height>16777215</height>
            </size>
           </property>
           <widget class="QLabel" name="label_brightness">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>60</width>
              <height>14</height>
             </rect>
            </property>
            <property name="text">
             <string>contrast</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QSlider" name="slider_brightness">
            <property name="geometry">
             <rect>
              <x>256</x>
              <y>16</y>
              <width>120</width>
              <height>22</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(250, 250, 250);</string>
            </property>
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QSlider" name="slider_contrast">
            <property name="geometry">
             <rect>
              <x>64</x>
              <y>16</y>
              <width>120</width>
              <height>22</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(250, 250, 250);</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>200</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="label_contrast">
            <property name="geometry">
             <rect>
              <x>192</x>
              <y>20</y>
              <width>60</width>
              <height>14</height>
             </rect>
            </property>
            <property name="text">
             <string>Brightness</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="pb_inverse_color">
            <property name="geometry">
             <rect>
              <x>382</x>
              <y>10</y>
              <width>31</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>/</string>
            </property>
            <property name="shortcut">
             <string>/</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TiffViewer.qrc"/>
 </resources>
 <connections/>
</ui>
